app/controllers/users_controller.rb

```diff
@@ -12,7 +12,9 @@ def index
       mentoring:  'Helping as a Mentor',
       organizing: 'Helping as an Organizer'
     }
-    @users = User.ordered(params[:sort], params[:direction]).group('users.id').with_all_associations_joined
+    @users = User.ordered(params[:sort],
+                          params[:direction]).group('users.id').with_all_associations_joined.
```

Comment:
Not sure, what's the reason for not using a scope here? Also, out of curiosity why is it limiting to 16 rows per page? It's a bit of an odd number (usually people stick with some multiple of 10).